index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main>
+        <nav class="nav-bar">
+            <ul>
+                <li><a href="#"> <img class="apple-img" src="images/apple.png"> </a></li>
```

Comment:
`SUGGESTION`

Consider adding an `alt` attribute since the image is a link to **apple home**.

Reply:
I am reading about that, now I know it's for a good quality. Suggestion done.